docs/socialmedia.md

```diff
@@ -0,0 +1,19 @@
+---
+title: socialmedia.md
+summary: Brief description of Blog platform, Twitter, Facebook, ...
+date: 2021-02-08  
+---
+
+# Social media
+
+## DietPi Blog
+
+T.b.d.
```

Comment:
Shall we remove the entry as long as were not sure we can finish it not too far future?

Reply:
Done.